From: Michael Vogt <michael.vogt@ubuntu.com>
Date: Wed, 25 May 2022 18:22:01 +0800
Subject: Allow symlinks for config/includes.chroot/etc/resolv.conf

Backport from the 4.0 branch to allow symlinks for 
config/includes.chroot/etc/resolv.conf so that a image can have a
symlink to /run/resolvconf/resolv.conf
---
 scripts/build/lb_chroot_resolv | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/build/lb_chroot_resolv b/scripts/build/lb_chroot_resolv
index 37bf8af..9a7a052 100755
--- a/scripts/build/lb_chroot_resolv
+++ b/scripts/build/lb_chroot_resolv
@@ -76,7 +76,7 @@ case "${1}" in
 		# Creating lock file
 		Create_lockfile .lock
 
-		if [ -e config/includes.chroot/etc/resolv.conf ]
+		if [ -e config/includes.chroot/etc/resolv.conf ] ||  [ -L config/includes.chroot/etc/resolv.conf ];
 		then
 			# Copying local resolv.conf
 			cp -a config/includes.chroot/etc/resolv.conf chroot/etc/resolv.conf
